### PR TITLE
issue #5

### PR DESCRIPTION
**Pull request by BilalAlmalih**
creating categories #5

### CODE REVIEW

**Review by wasimtayyan — APPROVED** [review]
Clean & easy to read code, I don't think there is anything to change

for the one who is going to merge this branch just take care of the Conflicting file 'page.js'

File: tsconfig.json
Comment (wasimtayyan) [inline tsconfig.json:1]: why do you want to use TypeScript

File: components/homePage-components/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import React from "react";
+import TrendingUpIcon from "@mui/icons-material/TrendingUp";
+import GroupsIcon from "@mui/icons-material/Groups";
+import GestureIcon from "@mui/icons-material/Gesture";
+import StorageOutlinedIcon from "@mui/icons-material/StorageOutlined";
+import DesignServicesOutlinedIcon from "@mui/icons-material/DesignServicesOutlined";
+import TipsAndUpdatesIcon from "@mui/icons-material/TipsAndUpdates";
+import AutoAwesomeIcon from "@mui/icons-material/AutoAwesome";
+
+const CategoriesSection = () => {
```
Comment (cerenss) [inline components/homePage-components/Categories.jsx:11]: For future usage, I think component should not be an arrow function. Might be useful to destructure and/or use props when necessary.

File: components/homePage-components/FeaturedCourse.jsx [hunk 2]
```diff
@@ -2,35 +2,35 @@
...
+              return (
+                <Course
+                  key={course.id}
+                  title={course.title}
+                  authorName={course.authorName}
+                  authorImage={course.authorImage}
+                  courseImage={course.courseImage}
+                  price={course.price}
+                  duration={course.duration}
+                  reating={course.reating}
```
Comment (cerenss) [inline components/homePage-components/FeaturedCourse.jsx:29]: Nit: typo 
reating --> rating

**Review by cerenss — CHANGES_REQUESTED** [review]
Nit: Reviews on typo and component arrow function.

File: tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  "include": [
+    "next-env.d.ts",
+    ".next/types/**/*.ts",
+    "**/*.ts",
+    "**/*.tsx"
+  ],
+  "exclude": [
+    "node_modules"
+  ]
+}
```
Comment (cerenss) [inline tsconfig.json:35]: Deploy error caused by TypeScript:

 It looks like you're trying to use **TypeScript** but **do not have the required package(s) installed.**
_11:50:25 AM:_ Please install @types/node by running:
_11:50:25 AM:_ 	**npm install --save-dev @types/node**
_11:50:25 AM:_ If you are not trying to use TypeScript, please **remove the tsconfig.json file from your package root** (and any TypeScript files in your pages directory).

The deploy log points out that if you are trying to use TypeScript you need run the specified command to install all necessary packages, if not remove the tsconfig.json file.

**Review by cerenss — CHANGES_REQUESTED** [review]

**Pull request closed without merging**